io.github.baldeuniversel.am-okay.yml

```diff
@@ -0,0 +1,48 @@
+id: io.github.baldeuniversel.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --filesystem=host
+    - --filesystem=home 
```

Comment:
`filesystem=home` is redundant since you're also specifying `filesystem=host`. 

Choose just one.

Reply:
Done !